server/graphql/v2/mutation/ExpenseMutations.ts

```diff
@@ -38,6 +39,42 @@ const expenseMutations = {
       });
     },
   },
+  editExpense: {
+    type: new GraphQLNonNull(Expense),
+    description: 'To update an existing expense',
+    args: {
+      expense: {
+        type: new GraphQLNonNull(ExpenseUpdateInput),
+        description: 'Expense data',
+      },
+    },
+    async resolve(_, { expense }, req): Promise<object> {
+      return editExpenseLegacy(req.remoteUser, {
+        id: idDecode(expense.id, IDENTIFIER_TYPES.EXPENSE),
+        description: expense.description,
+        tags: expense.tags,
+        type: expense.type,
+        privateMessage: expense.privateMessage,
+        invoiceInfo: expense.invoiceInfo,
+        amount: expense.attachments?.reduce((total, att) => total + att.amount, 0),
+        PayoutMethod: expense.payoutMethod && {
+          id: expense.payoutMethod.id && idDecode(expense.payoutMethod.id, IDENTIFIER_TYPES.PAYOUT_METHOD),
+          data: expense.payoutMethod.data,
+          name: expense.payoutMethod.name,
+          isSaved: expense.payoutMethod.isSaved,
+          type: expense.payoutMethod.type,
+        },
+        attachments: expense.attachments?.map(attachment => ({
+          id: attachment.id && idDecode(attachment.id, IDENTIFIER_TYPES.EXPENSE_ATTACHMENT),
+          url: attachment.url,
+          amount: attachment.amount,
+          incurredAt: attachment.incurredAt,
+          description: attachment.description,
+        })),
+        fromCollective: null, // TODO payee
```

Comment:
Will be tackled with https://github.com/opencollective/opencollective/issues/2266